compiler/optimizations/optimizeOnClauses.cpp

```diff
@@ -147,22 +178,27 @@ isFastPrimitive(CallExpr *call, bool isLocal) {
     // If this test is true, a remote get is required.
     if (!(call->get(1)->typeInfo()->symbol->hasFlag(FLAG_WIDE_REF) &&
           call->get(1)->getValType()->symbol->hasFlag(FLAG_WIDE_CLASS))) {
-      DEBUG_PRINTF(" *** OK (PRIM_WIDE_GET_LOCALE, etc.): %s\n",
-                   call->primitive->name);
-      return true;
+      return FAST_AND_LOCAL;
     }
-    break;
+    return FAST_NOT_LOCAL;
+
+  case PRIM_ARRAY_SHIFT_BASE_POINTER:
```

Comment:
Is this required to get _downEndCount to run in the AM or is it orthogonal to that?